### PR TITLE
Premium fixes

### PR DESCRIPTION
**Pull request by sbrossie**
Work on toc for make it more obvious that we have premium capabilities:
* Included a new Premium Capability intro page
* Reordered pages to have premium at the bottom (right before `References`)
* Added missing banners
* Added a purple dot for premium pages (to make it obvious from the toc)
* Moved aws doc all under aws folder.

This is what this looks like:

Screenshot 2024-12-07 at 11 55 07 AM

### CODE REVIEW

File: html5/_main_toc.html.slim [hunk 1]
```diff
@@ -2,7 +2,8 @@ nav.sidebar-nav
   ul.nav.navbar-nav
     li
       .icon-title
-        a.bd-toc-link.main-link role="button" Getting Started
```
Comment (pierre) [inline html5/_main_toc.html.slim:5]: Verified that on preview pages there is no issue with the new spacing.

**Review by pierre — COMMENTED** [review]
I would suggest to merge @reshmabidikar's PR first, I'm worried about AWS related conflicts.

FYI I'm not seeing the new Premium Capability page? 🤔

**Comment by sbrossie** [comment]
> I'm not seeing the new Premium Capability page? 

Forgot to push 😮‍💨 

Fixed in [2730507](https://github.com/killbill/killbill-docs/pull/581/commits/27305072c1a1dbf383d510edca812c723fcbc83f)

File: html5/_main_toc.html.slim (hunk 1 above)
Comment (sbrossie) [reply html5/_main_toc.html.slim:5]: Yep, from what I am seeing this works fine.

**Comment by sbrossie** [comment]
> I would suggest to merge @reshmabidikar's PR first, I'm worried about AWS related conflicts.

Yeah I had not realized there was draft PR with AWS changes...

**Comment by sbrossie** [comment]
Closing this PR as I recreated it [here](https://github.com/killbill/killbill-docs/pull/582) (- aws files move in [804396d](https://github.com/killbill/killbill-docs/pull/581/commits/804396deb7097083618a5085bd4909998cc5a6cc)

**Pull request closed without merging**